### PR TITLE
Release notes v1.2.0

### PR DESCRIPTION
**Pull request by irenebm**

### CODE REVIEW

File: docs/rst/notes/notes.rst [hunk 1]
```diff
@@ -7,24 +7,20 @@
...
-* New :ref:`user_manual_user_interface_log_verbosity_argument` and :ref:`user_manual_user_interface_log_filter_argument`
-  arguments to configure *DDS Router* and *Fast DDS* logs shown in application.
-
 This release includes the following **CI improvements**:
 
-* Add *Address Sanitizer* check to all tests.
-* Add *Thread Sanitizer* check to all tests.
-* Compile with `-Wall` flag all CI jobs.
+* Add blackbox tests for repeater in WAN case.
+* Add unittest tests for ``PayloadPool``, ``FastPayloadPool`` and ``MapPayloadPool`` clases.
```
Comment (jparisu) [inline docs/rst/notes/notes.rst:16]: Are these tests in this release?

**Review by jparisu — CHANGES_REQUESTED** [review]

File: docs/rst/notes/notes.rst (hunk 1 above)
Comment (irenebm) [reply docs/rst/notes/notes.rst:16]: They were added in #336 but the tests for MapPayloadPool and FastPayloadPool were just edited so I decided to remove them.

File: docs/rst/notes/notes.rst [hunk 2]
```diff
@@ -13,7 +13,7 @@ Version v1.2.0
 This release includes the following **CI improvements**:
 
 * Add blackbox tests for repeater in WAN case.
-* Add unittest tests for ``PayloadPool``, ``FastPayloadPool`` and ``MapPayloadPool`` clases.
+* Add unittest tests for ``PayloadPool`` class.
```
Comment (jparisu) [inline docs/rst/notes/notes.rst:16]: This is still not correct I think. No tests were added

**Review by jparisu — CHANGES_REQUESTED** [review]
small datum

**Review by jparisu — APPROVED** [review]
LGTM

**Pull request merged**